src/backend/joanie/core/flows/order.py

```diff
@@ -235,6 +235,8 @@ def init(self, billing_address=None):
                 )
 
         self.instance.freeze_target_courses()
+        if not self.instance.is_free and self.instance.has_contract:
+            self.instance.generate_schedule()
```

Comment:
This is not true anymore. It should be generated everytime a signing process is started

Reply:
OK, I'll change that.